net/mwan3/files/etc/hotplug.d/iface/16-mwan3-user

```diff
@@ -16,7 +16,6 @@
 
 	config_get_bool enabled "$INTERFACE" enabled 0
 	[ "${enabled}" -eq 1 ] || {
```

Comment:
Can also now remove braces here

Reply:
I will change this also

Reply:
I looked this up again in the source of other packages.
There is no uniformity.
There are different possibilities.
I do not know what is right or wrong.
What does the shellcheck say?

Reply:
I don't think shellcheck has an opinion on this. In general if it is one line, I prefer no braces, if it is a few lines, I like to use braces, and if it is many lines or needs an 'else', I prefer to use an 'if ; then; fi' statement. 

I don't have a strong preference, so it is fine with me to leave as is if you prefer.